wiki/osu!supporter/en.md

```diff
@@ -1,61 +1,64 @@
 ---
-needs_cleanup: true
+tags:
+  - support
+  - supporter
 ---
 
-<!-- TODO: website updates, needs a review on the writing style -->
-
 # osu!supporter
 
-*For the osu!supporter page from the website, see: [support the game](https://osu.ppy.sh/home/support)*
-
-## Benefits
+The **osu!supporter** is a *temporary* title given to the players who supported osu! by making a voluntary, monetary donation to help it thrive. Supporters receive a handful of cosmetical and technical benefits, most of which are active only when the title is present, such as:
 
-*For a list of goodies from having osu!supporter, see: [support the game](https://osu.ppy.sh/home/support)*
+- Distinctive bright yellow nickname in the [chat](/wiki/Client/Interface/Chat_console);
+- Access to osu!direct, the in-game beatmap search and download engine;
+- Relaxed limits on number of favourited or uploaded beatmaps;
+- Ability to additionally skin osu!;
+- Custom [leaderboards](/wiki/Beatmap#leaderboards).
```

Comment:
```suggestion
- Custom [leaderboards](/wiki/Beatmap#leaderboards).
- Increased number of friends that you can add
```
In the promo page:

> More Friends
> The maximum number of friends you can have is increased from 250 → 500

Reply:
I'm not sure if unlimited download in beatmap listing is still a feature for osu!supporter only or it's already available for all

Reply:
```csharp
BEATMAPSET_USER_DOWNLOAD_LIMIT_HOURLY=200
BEATMAPSET_USER_DOWNLOAD_LIMIT_HOURLY_SUPPORTER=800
```

Reply:
not going to write these in as-is, but thanks for the confirmation